packages/lesswrong/server/manualMigrations/2023-09-08-filterDebateResponses.ts

```diff
@@ -0,0 +1,12 @@
+import { registerMigration } from './migrationUtils';
```

Comment:
Old man Habryka too lazy to learn how to write automatic migrations

Reply:
lol, I would actually like to learn about how to do this automatically. Will figure it out next week.